packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -49,6 +49,9 @@ export type TSClientOptions = O.Required<GenerateClientOptions, 'runtimeBase'> &
   reusedTs?: string // the entrypoint to reuse
   /** When js doesn't need to be regenerated */
   reusedJs?: string // the entrypoint to reuse
+
+  /** result of getEnvPaths call */
+  envPaths: EnvPaths
```

Comment:
Added to generator options to avoid adding dependency between `@prisma/client` and `@prisma/prisma-schema-wasm`